### DISCUSSION TITLE
Snapshot Release Notes

### DISCUSSION THREAD

**Discussion by Stephan-P** [keepassxreboot/keepassxc#10051] [Ideas]
As v. 2.8 is in the making, I regularly check https://snapshot.keepassxc.org to find at times that a new snapshot has been issued. 
But installing the next snapshot is something of a gamble as there's no information included.

Would it be possible to add a brief Release Notes document to a Snapshot?
E.g.
- Added:
- Fixed:
- Please test/check:

**Comment by droidmonkey** [comment]
Recommend bookmarking this link: https://github.com/keepassxreboot/keepassxc/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc+milestone%3Av2.8.0

It'll show the latest PRs merged into develop branch
